DEPRECATIONS.md

```diff
@@ -10,6 +10,10 @@ Deprecations will be tracked by the release they are announced in, and then upda
 ## Deprecation History
 All deprecations are listed below, with the most recent announcements at the top.
 
+### Lagoon v2.18.0
+release link: https://github.com/uselagoon/lagoon/releases/tag/v2.18.0
+* The standard drupal based tasks that Lagoon ships with (drush ....) have been flagged as deprecated and should not be used anymore. These will need to be replaced with [custom tasks](https://docs.lagoon.sh/using-lagoon-advanced/custom-tasks/). Example replacement tasks will be provided when these are completely deprecated.
```

Comment:
Maybe this?
```
Example replacement tasks will be provided prior to their removal
```

Reply:
This way the wording makes it sound like we aren't going to wait til the end to tell people how to replace them